openssl_patches/1.1.1b/tests/mksmime_certs.txt_111b.patch

```diff
@@ -0,0 +1,108 @@
+diff --git a/test/smime-certs/mksmime-certs.sh b/test/smime-certs/mksmime-certs.sh
+index c98e164b..8fad96c2
+--- a/test/smime-certs/mksmime-certs.sh
++++ b/test/smime-certs/mksmime-certs.sh
+@@ -9,76 +9,75 @@
+ 
+ # Utility to recreate S/MIME certificates
+ 
+-OPENSSL=../../apps/openssl
```

Comment:
Leave the OPENSSL declaration in.
Make it equal to openssl. We may want to change this in the future.

Reply:
Ok.  Not sure why I didn't think of that.  I must have been tired.